lib/livebook/notebook/explore/intro_to_vega_lite.livemd

```diff
@@ -1,3 +1,960 @@
-# Plotting with VegaLite and Kino
+# Plotting with VegaLite
 
-TODO: content 🐈
+## Setup
+
+```elixir
+Mix.install([
+  {:kino, "~> 0.1.0"},
+  {:vega_lite, "~> 0.1.0"},
+  {:jason, "~> 1.2"}
+])
```

Comment:
Can you add line about each library and what they are about?

Reply:
Do we even use Jason in any of these examples?

Reply:
Good catch, I wonder if we should have an example with `Lv.from_json`, but perhaps a bunch of examples is sufficient here?

Reply:
Nah, it is fine as is IMO. :)